sdk/push-fcm/src/main/java/com/klaviyo/pushFcm/KlaviyoNotification.kt

```diff
@@ -104,7 +106,7 @@ class KlaviyoNotification(private val message: RemoteMessage) {
 
         NotificationManagerCompat
             .from(context)
-            .notify(generateId(), notification.build())
+            .notify(message.notificationTag, generateId(), notification.build())
```

Comment:
Needs fixing: `generateId()` returns the time in milliseconds. `notify` dedupes based on tag _and_ id, so because ID is already (pretty much) unique, this will never dedupe. Instead we should just use a constant value for ID (like 0), and just rely on tag. Caveat -- we'd need to autogenerate a tag (timestamp should suffice) if `notificationTag` is null.

Reply:
BTW I recall this worked in my original testing, but I think thats because I was sending the notifications near simultaneously from my local, so the device probably did `generateId` twice within a millisecond!